tests/unit/test_client_errors.py

```diff
@@ -0,0 +1,49 @@
+import json
+
+import httpx
+import pytest
+import respx
+from respx import MockRouter
+
+from apify_client._errors import ApifyApiError
+from apify_client._http_client import HTTPClient, HTTPClientAsync
+
+_TEST_URL = 'http://example.com'
+_EXPECTED_MESSAGE = 'some_message'
+_EXPECTED_TYPE = 'some_type'
+_EXPECTED_DATA = {
+    'invalidItems': {'0': ["should have required property 'name'"], '1': ["should have required property 'name'"]}
+}
+
+
+@respx.mock
+@pytest.fixture
```

Comment:
Is this autoused somehow?

Reply:
It is used by 
@pytest.mark.usefixtures('mocked_response')

I would normally just pass the fixture into the args, but then ruff is complaining about unused args :-(, so I thought this somewhat less common way of using fixtures is better than #ignore comment.

Reply:
I would prefer either fixture(autouse=True) or disabling that rule for tests